static/js/script.js

```diff
@@ -530,3 +530,10 @@ $(function() {
         $('#hero').addClass('no-sub');
     }
 });
+
+var url = "https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=kubernetes";
```

Comment:
We should probably place a guard condition that this snippet should only run if `window.location.url` is the correct path? That way we don't impact performance

Reply:
hello @iamneha, not sure I understand what this page is supposed to display -- the result of the query above embedded in a k8s docs page? How will you handle a large result or no result? Would it be possible to just provide a link to the mitre page instead, or have I completely misunderstood your intention?

Reply:
The response will be unreadable here because of the Same Origin Policy.

+1 for just a link instead.

Reply:
@kbhawkey @captn3m0 
You are correct we can simply put the link but as per the discussion in the #sig-docs-security group. We have decided to scrap the website and show the result in table on the website.

> hello @iamneha, not sure I understand what this page is supposed to display -- the result of the query above embedded in a k8s docs page? How will you handle a large result or no result? Would it be possible to just provide a link to the mitre page instead, or have I completely misunderstood your intention?

@kbhawkey before handling large result or no result I want to show the result on table. I am not frontend person so I don't know how to show this response in table specially in `.md` file. In `.html` file I am able to show but not sure about `.md` file. If you know something here please help me :)

Reply:
@iamneha, I would suggest writing a simple python script to pull the data and then display the contents, statically. The script could be run during k8s release time or upon demand. Is there already a list of security faults recorded as part of the k8s/k8s repo? How is the data reported to mitre?